lightningd/lightningd.c

```diff
@@ -197,7 +197,7 @@ static struct lightningd *new_lightningd(const tal_t *ctx)
 	 * NULL.  So we start with a zero-length array. */
 	ld->proposed_wireaddr = tal_arr(ld, struct wireaddr_internal, 0);
 	ld->proposed_listen_announce = tal_arr(ld, enum addr_listen_announce, 0);
-	ld->portnum = DEFAULT_PORT;
+	// ld->portnum moved after handle_early_opts
```

Comment:
I think if it is only a comment, we can remove it. Usually the comment describe somethings, and a refactoring it is not considered "somethings" :)

Reply:
OK, fair enough. Will remove it tomorrow. Thanks for feed-back.

Reply:
Done.